Samples/RenderGraph/RenderGraphViewer/RenderGraphViewer.cpp

```diff
@@ -41,18 +47,55 @@ void RenderGraphViewer::onShutdown(SampleCallbacks* pSample)
 void RenderGraphViewer::onLoad(SampleCallbacks* pSample, const RenderContext::SharedPtr& pRenderContext)
 {
     // if editor opened from running render graph, get the name of the file to read
-    std::vector<ArgList::Arg> commandArgs = pSample->getArgList().getValues("tempFile");
-   
+    mDefaultSceneName = gkDefaultScene;
+
```

Comment:
add a function to parse command line args to clean up . -scene instead of defaultScene -scene defaultImage -image .  -editor for editor running.